app/Controllers/feedController.php

```diff
@@ -415,7 +415,7 @@ public static function actualizeFeed($feed_id, $feed_url, $force, $simplePiePush
 				$entryDAO->commit();
 			}
 
-			if ($feed->hubUrl() && $feed->selfUrl()) {	//selfUrl has priority for WebSub
+			if ($pubsubhubbubEnabledGeneral && $feed->hubUrl() && $feed->selfUrl()) {	//selfUrl has priority for WebSub
```

Comment:
```suggestion
			if ($feed->pubSubHubbubEnabled() && $feed->hubUrl() && $feed->selfUrl()) {	//selfUrl has priority for WebSub
```

I think `$feed->pubSubHubbubEnabled()` returns `false` when `$pubsubhubbubEnabledGeneral` is `false` and I find it's more logical this way

Reply:
`$feed->pubSubHubbubEnabled()` would not work, but you are right that it was not the correct variable :-)
Fix coming using 
https://github.com/FreshRSS/FreshRSS/blob/1ddf98c50b2e0388c76386202bd8be3442640418/app/Controllers/feedController.php#L279

Reply:
Reverted to the first version https://github.com/FreshRSS/FreshRSS/pull/2922